docs/StardustDocs/topics/KPropertiesApi.md

```diff
@@ -2,7 +2,8 @@
 
 <!---IMPORT org.jetbrains.kotlinx.dataframe.samples.api.ApiLevels-->
 
-`Kotlin Dataframe` can be used as an intermediate structure for data transformation between two data formats. If either source or destination is a Kotlin object, e.g. data class, it is convenient to use its properties for typed data access in `DataFrame`.
+[`DataFrame`](DataFrame.md) can be used as an intermediate structure for data transformation between two data formats. 
```

Comment:
Maybe for later: This sentence is phrased like we're talking about the library "DataFrame" instead of "A DataFrame", the object. This is why "the" is so important :).

Reply:
This happens all over the place btw. Job for the tech writers :)